logstash-core-event-java/src/test/java/com/logstash/KeyNodeTest.java

```diff
@@ -0,0 +1,27 @@
+package com.logstash;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+
+public class KeyNodeTest {
```

Comment:
I propose we add the check in our `join` implementation to use an empty string when the item is null. WDYT?